vcluster/cli/vcluster_delete.md

```diff
@@ -27,10 +27,10 @@ vcluster delete test --namespace test
 ## Flags
 
 ```
-      --auto-delete-namespace   If enabled, vcluster will delete the namespace of the vcluster if it was created by vclusterctl. In the case of multi-namespace mode, will also delete all other namespaces created by vcluster (default true)
+      --auto-delete-namespace   If enabled, vcluster will delete the namespace of the vcluster if it was created by vclusterctl. In the case of namespace mode, will also delete all other namespaces created by vcluster (default true)
```

Comment:
Maybe instead of calling it "namespace mode" we'd say "In case of enabled sync.toHost.namespaces" or "in case vcluster is using namespace syncing"? Wdyt?